src/index.html

```diff
@@ -1,16 +1,41 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search">
+      <form class="search-bar">
+        <img 
+          class="search-svg search-svg--big" 
+          alt="images" 
+          src="images/Search.svg"
+          >
```

Comment:
``
```suggestion
        <img 
          class="search-svg search-svg--big" 
          alt="images" 
          src="images/Search.svg"
        >
```
pay attention to formatting